android/src/paper/java/com/reactnativecommunity/asyncstorage/NativeAsyncStorageModuleSpec.java

```diff
@@ -0,0 +1,52 @@
+
+/**
+ * This code was generated by [react-native-codegen](https://www.npmjs.com/package/react-native-codegen).
+ *
+ * Do not edit this file as changes may cause incorrect behavior and will be lost
+ * once the code is regenerated.
+ *
+ * @generated by codegen project: GenerateModuleJavaSpec.js
+ *
+ * @nolint
+ */
```

Comment:
Updated the comment in https://github.com/react-native-async-storage/async-storage/pull/910/commits/0d41f20e836a2bbcd03acda409759550333c31fa.

`paper` because that's the name of the renderer used on the old architecture. In hindsight, `oldarch` might've been better. Should I change to that?

Reply:
TIL!
Yes please, I think it'd be cleaner 🙏

Reply:
Done in https://github.com/react-native-async-storage/async-storage/pull/910/commits/f4edc00759789e5a84096326dc13aaf7887a70fe 🎉